emission/core/config.py

```diff
@@ -0,0 +1,44 @@
+import json
+import logging
+import os
+
+def get_config_data_from_env():
+    config_data_env = {
+        "timeseries": {
+            "url": os.getenv('DB_HOST', "localhost"),
+            "result_limit": os.getenv('DB_TS_RESULT_LIMIT', 250000)
+        }
+    }
+    return config_data_env
+
+def check_unset_env_vars():
+    config_data_env = {
+        "timeseries": {
+            "url": os.getenv('DB_HOST'),
+            "result_limit": os.getenv('DB_TS_RESULT_LIMIT')
+        }
+    }
+    return not any(config_data_env.values())
+
+def get_config_data():
+    try:
+        config_file = open('conf/storage/db.conf')
```

Comment:
I also don't think we need this given that we are going to use the environment variable going forward and it falls back to `localhost` `"url": os.getenv('DB_HOST', "localhost"),`

Reply:
we should also remove the sample file to avoid confusion since we don't use it. Note that before this change, we used to fall back to the sample file if the `db.conf` file did not exist, but we have removed that now, so it is not clear what benefit the sample file gives us in terms of working out of the box or anything

Reply:
Our logic here was that we wanted to maintain the functionality of db.conf for users running the server locally, in case they have it set in a certain way.

Reply:
I would be fine with retaining this for now as a backwards compat hack but it should be removed in the next release.